src/main/java/org/jvnet/hudson/plugins/repositoryconnector/Artifact.java

```diff
@@ -73,6 +75,17 @@ public String getTargetFileName() {
         return targetFileName;
     }
 
+    /**
+     * @return the uploadPOM
+     */
+    public boolean getUploadPOM() {
+        // Backwards compatibility: if null, config not updated: fallback to true
```

Comment:
This part here I was unsure of. Using "positive" booleans (ex: not have "disable = true") caused a bit of a backwards compatibility gotcha in terms of the value not being set in previous versions, reason for the class var being Boolean.
Open to suggestions on how to accomplish this more elegantly.